src/main/java/decodes/tsdb/OracleDbmsSpecificFunctions.java

```diff
@@ -34,28 +33,24 @@ public void setSessionTimeZone(String tzName)
 		throws DbIoException
 	{
 		String q = null;
-		Statement st = null;
 
+		DaiBase dao = new DaoBase(theDb, "PgDbmsSpecificFunctions");
```

Comment:
The class is "OracleDbms..." but this string says "PgDbms.."

Reply:
Done